spec/marketing_version_calculator_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+describe Fastlane::Wpmreleasetoolkit::Versioning::MarketingVersionCalculator do
+  describe 'calculates the next release version when using marketing versioning' do
+    context 'when the minor version is not 9' do
+      it 'increments the minor version when the minor version is less than 9' do
+        version = Fastlane::Models::AppVersion.new(13, 5, 1, 1)
+        bumped_version = described_class.new.next_release_version(after: version)
+        expect(bumped_version.to_s).to eq('13.6.0.0')
+      end
+
+      it 'increments the minor version when the minor version is greater than 9' do
+        version = Fastlane::Models::AppVersion.new(13, 11, 1, 1)
+        bumped_version = described_class.new.next_release_version(after: version)
+        expect(bumped_version.to_s).to eq('13.12.0.0')
+      end
```

Comment:
Shouldn't this kind of case roll over to next major instead, because `minor>9` already?

Personally I'd instead expect:
```suggestion
      it 'increments the major version when the minor version is greater than 9' do
        version = Fastlane::Models::AppVersion.new(13, 11, 1, 1)
        bumped_version = described_class.new.next_release_version(after: version)
        expect(bumped_version.to_s).to eq('14.0.0.0')
      end
```
Alternatively, we could also consider `UI.user_error!` on such a case, stating that `MarketingVersionCalculator` does not allow minor versions to be >9 so the provided `AppVersion` is invalid in that scheme… 🤷

Reply:
Addressed in https://github.com/wordpress-mobile/release-toolkit/pull/512/commits/f18258feed621c09161ee2c9dc180ba8d1517199

I went with raising a `UI.user_error!` if it's more than 9. To be able to easily calculate the previous version number, I think we want to be strict about the maximum of 9 for the minor number.